### ISSUE TITLE
projectmgr-nvim crash

### ISSUE THREAD

**Issue by MrMic** [AstroNvim/astrocommunity#765]
### Checklist

- [X] I have searched through the AstroNvim docs
- [X] I have searched through the existing issues of AstroNvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.9.4

### Operating system/version

Linux Ubuntu 20.04

### Terminal/GUI

kitty

### Describe the bug

Press required keys `<leader>P` and got this:
```
E5108: Error executing lua ...vim/lazy/projectmgr.nvim/lua/projectmgr/file_adapter.lua:36: Expected value but found T_END at character 1
stack traceback:
	[C]: in function 'read_db'
	...vim/lazy/projectmgr.nvim/lua/projectmgr/file_adapter.lua:36: in function 'get_projects'
	...lazy/projectmgr.nvim/lua/projectmgr/telescope_picker.lua:14: in function 'show_telescope'
	...lazy/projectmgr.nvim/lua/projectmgr/telescope_picker.lua:88: in function 'open_window'
	[string ":lua"]:1: in main chunk
```

### Steps to Reproduce

See explanations above

### Expected behavior

Open the Telescope and show projects

**Comment by Uzaaft** [comment]
@MrMic  The plugin spec in astrocommunity works fine for me in

**Comment by MrMic** [comment]
Yes, for me too: works fine a few weeks ago, but when I try to use it again
recently I got this error ...

**Comment by Uzaaft** [comment]
It´ll be tough for us to debug this, when we can´t reproduce this. 
I tried on a ubuntu vm as well, same story there.

**Comment by Uzaaft** [comment]
Try and wipe your cache and lazy cache, and see if that won´t fix it.

**Comment by MrMic** [comment]
How to do this?

**Comment by Uzaaft** [comment]
```bash
rm -rf ~/.cache/nvim
rm -rf ~/.local/share/nvim
rm -rf ~/.local/state/nvim
```

**Comment by Uzaaft** [comment]
I think your best bet is to open an issue at the plugin repo. This doesnt seem like an astro issue.

**Comment by MrMic** [comment]
Yes, I think so.

**Comment by MrMic** [comment]
I deleted and reinstalled the plugin, and this 
solve my issue ... weird.
Sorry for the noise.
Rgds,
